src/open_inwoner/cms/cases/tests/test_htmx.py

```diff
@@ -432,7 +432,7 @@ def test_cases(self, m):
         upload_form = page.locator("#document-upload")
         expect(upload_form).to_be_visible()
 
-        file_input = upload_form.get_by_text(_("Sleep of selecteer bestand"))
+        file_input = page.locator("[name='file']")
```

Comment:
I think we should prefer the [get_by_label](https://playwright.dev/python/docs/api/class-page#page-get-by-label) where possible (or any other selector that testes a11y).

Reply:
I changed the `id` and `for` attributes for the `input` and `label` elements to make the test work with `get_by_label`. Let me know if this is not desirable.

Reply:
I did do a small change, instead of using a hardcoded id I used the "auto_id" for both the input and the label.